index.js

```diff
@@ -1,85 +1,9 @@
-'use strict';
+'use strict'
```

Comment:
Missing `;`

```suggestion
'use strict';
```

Reply:
Thank you for checking it, on the move now, will send an update soon